docs/source/configure/index.rst

```diff
@@ -146,6 +148,54 @@ Environment-specific parameters
   ``"initial_peers" : [{"address":"127.0.0.1:10001", "public_key":
   "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"}]``
 
+Alternative
```

Comment:
I would avoid marking this possibility as "alternative", which sounds like "secondary". The approach with environment is meant to provide completely the same functionality as the config file, so IMO they should be explained as equally adequate approaches. What about making two same-level sections, where one describes config file syntax, and the other is about env vars?

Reply:
Ok, will do!